### PR TITLE
Give a better error when wrapping on search

### PR DESCRIPTION
**Pull request by Carpetsmoker**
Previously, it just said "Text not found" when you hit the bottom.

### CODE REVIEW

File: qutebrowser/browser/webview.py [hunk 1]
```diff
@@ -487,12 +487,25 @@ def search(self, text, flags):
         old_scroll_pos = self.scroll_pos
         flags = QWebPage.FindFlags(flags)
         found = self.findText(text, flags)
+        backward = int(flags) & QWebPage.FindBackward
```
Comment (The-Compiler) [inline qutebrowser/browser/webview.py]: Is the `int()` here needed? At least below, `flags & QWebPage.HighlightAllOccurrences` seems to work fine.

**Comment by The-Compiler** [comment]
Clarification: This seems only for `general -> wrap-search` set to `False` - with `wrap-search` on, it seems to work as expected.

File: qutebrowser/browser/webview.py (hunk 1 above)
Comment (Carpetsmoker) [inline qutebrowser/browser/webview.py]: I don't know why the `int` is here, I just moved the line ...

Comment (The-Compiler) [inline qutebrowser/browser/webview.py]: Oh, whoops, sorry!

**Pull request merged**